### ISSUE TITLE
error in typescript express

### ISSUE THREAD

**Issue by rabpeter** [sebbo2002/ical-generator#121]
when I call cal.serve(res), I'm getting this issue. 

```
[ts] Expected 2 arguments, but got 1.
index.d.ts(152, 32): An argument for 'filename' was not provided.
```

**Comment by sebbo2002** [comment]
Seems to be a bug in the typescript definition…

**Comment by ivanbreet** [comment]
Hi, we are also getting some TypeScript related errors:
```
ERROR in ../node_modules/ical-generator/index.d.ts
(152,22): error TS2304: Cannot find name 'Response'.

ERROR in ../node_modules/ical-generator/index.d.ts
(258,3): error TS2666: Exports and export assignments are not permitted in module augmentations.
```

**Comment by evelynhathaway** [comment]
@ivanbreet I had the same issues and is now fixed in `v1.4.2` 😄
